src/main/resources/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpoint/help-uri.html

```diff
@@ -1,5 +1,5 @@
 <div>
-    URI to the Docker server you are using.
-    May be left blank to use the Docker default
-    (typically <code>unix:///var/run/docker.sock</code>).
+    URI to the Docker Host you are using.
+    May be left blank to use the Docker default (defined by DOCKER_HOST environment variable)
+    (typically <code>unix:///var/run/docker.sock</code> or <code>tcp://127.0.0.1:2376</code>).
```

Comment:
But it's our job to make UX as smooth as possible and provide useful help 
changed port in 6537090